client/blocks/inline-help/popover.jsx

```diff
@@ -119,13 +120,23 @@ class InlineHelpPopover extends Component {
 	};
 
 	switchToClassicEditor = () => {
-		const { siteId, optOut, classicUrl } = this.props;
-		optOut( siteId, classicUrl );
+		const { siteId, onClose, optOut, classicUrl } = this.props;
+
+		const proceed =
+			typeof window !== 'undefined'
+				? window.confirm( __( 'Are you sure you wish to leave this page?' ) )
+				: true;
```

Comment:
nice ternary, but in this case we can also use a simpler form…

```js
const proceed = typeof window === 'undefined' || window.confirm( … );
```

just noting because the ternary is maybe more complexity than we need to debug here 😉